cli/main.rs

```diff
@@ -1442,10 +1442,14 @@ fn unwrap_or_exit<T>(result: Result<T, AnyError>) -> T {
   match result {
     Ok(value) => value,
     Err(error) => {
+      let error_string = match error.downcast_ref::<JsError>() {
+        Some(e) => format_js_error(e),
+        None => format!("{:?}", error),
+      };
```

Comment:
Why is it needed?

Reply:
The only places errors are pretty printed are `cli/main.rs`, `cli/tools/test.rs` and `runtime/web_worker.rs`. This is where we apply `format_js_error()`. In fact, the `WorkerOptions::format_js_error_fn` in runtime is only for the third one.